static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -571,8 +571,6 @@ function WidgetViewerModal(props: Props) {
         }
         return (
           <ReleaseWidgetQueries
-            api={api}
-            organization={organization}
             widget={tableWidget}
             selection={modalSelection}
```

Comment:
I'm assuming `modalSelection` is different then the global page filters so we can't use the `usePageFilters` hook and omit this prop?